addons/mail/static/src/components/chatter_container/chatter_container.js

```diff
@@ -32,23 +32,21 @@ export class ChatterContainer extends Component {
     setup() {
         useModels();
         super.setup();
-        this.chatter = undefined;
-        this.chatterId = getChatterNextTemporaryId();
+        this.localChatter = undefined;
         this._insertFromProps(this.props);
-        onWillUpdateProps(nextProps => this._willUpdateProps(nextProps));
-        onWillDestroy(() => this._onWillDestroy());
+        onWillUpdateProps(nextProps => {
+            this._insertFromProps(nextProps);
+        });
+        onWillDestroy(() => this.deleteLocalChatter());
     }
 
-    _willUpdateProps(nextProps) {
-        this._insertFromProps(nextProps);
+    get chatter() {
```

Comment:
Ok as fix (minimal code change while fixing the issue)

Not to change in this PR: integration with `WebClientView` code should just insert data in models and let UI reflect modelling in a simple way that scales properly for all `Container`.